tradesman/model_creation/zoning/zones_with_location.py

```diff
@@ -29,11 +37,11 @@ def zones_with_location(hexb, all_subdivisions):
     empties = data_complete.division_name.isna()
 
     for idx, record in data_complete[empties].iterrows():
-        geo = record.geometry
-        dscrt = [x for x in dindex.nearest(geo.bounds, 10)]
-        dist = [states.loc[d, "geometry"].distance(geo) for d in dscrt]
+        geo = box(*record.geometry.bounds)
+        dscrt = [x for x in dindex.nearest(geo, 10)]
+        dist = [states.loc[d, "geometry"].distance(geo).tolist() for d in dscrt]
         m = dscrt[dist.index(min(dist))]
-        data_complete.loc[idx, "division_name"] = states.loc[m, "division_name"]
+        data_complete.loc[idx, "division_name"] = states.loc[m, "division_name"].tolist()[0]
```

Comment:
The correct syntax is 

`data_complete.loc[idx, "division_name"] = states.loc[m, "division_name"].values[0]`